test/recipes/95-test_external_pyca_data/cryptography.sh

```diff
@@ -48,13 +48,26 @@ pip install -e vectors
 echo "------------------------------------------------------------------"
 echo "Building cryptography"
 echo "------------------------------------------------------------------"
-CFLAGS="-I$O_BINC -I$O_SINC -L$O_LIB" pip install .
+LDFLAGS="-L$O_LIB" CFLAGS="-I$O_BINC -I$O_SINC -L$O_LIB" pip install .
 
 echo "------------------------------------------------------------------"
 echo "Running tests"
 echo "------------------------------------------------------------------"
 
-CFLAGS="-I$O_BINC -I$O_SINC -L$O_LIB" pytest -n auto tests --wycheproof-root=../wycheproof
+openssl version
+pwd
+find .. -name libcrypto.so
+find / -name libcrypto.so
+sudo rm /usr/lib/x86_64-linux-gnu/libcrypto.so
+sudo rm /usr/lib/x86_64-linux-gnu/libssl.so
```

Comment:
Desperation setting in? :disappointed:

Reply:
Wanted to see what it does.. This might be too late to do it though..

Reply:
This is popping up:
```
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.9.14/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.9.14/x64/lib
```
if there is a libcrypto in the latter, it would explain things...

I suspect it's not related.